### DISCUSSION TITLE
SDG_GOAL

### DISCUSSION THREAD

**Discussion by SandraMarq** [open-sdg/open-sdg#1954] [Q&A] [answered]
Hello.

When putting in the SDG_GOAL tag from 10, only the first number is displayed, why does that happen?

![image](https://user-images.githubusercontent.com/118024145/224789931-580634ed-37cd-4c99-b4af-457c28d08c46.png)

![image](https://user-images.githubusercontent.com/118024145/224790008-b7d442fd-e5b1-4523-98d1-492951995564.png)

Thanks

**Comment by sarah-beasley** [comment, marked as answer]
Hi, this is a problem with the way that markdown formats numbers. To format a numbered list in markdown, you just have to write any number at the start of the line e.g.:
12.
12.
12.
and it will know to make this actually appear as:
1.
2.
3.

Therefore, by writing '17.', it thinks you are just trying to write a list, so that's why it only appears as '1.' in your metadata. To change this, you could try writing something in front of the number, e.g. Goal 17. This should remove the numbered list formatting and just show the text you'd like instead.
